src/block-management/block-toolbar.js

```diff
@@ -36,45 +36,43 @@ export class BlockToolbar extends Component<PropsType> {
 		} = this.props;
 
 		return (
-			<SafeAreaView>
```

Comment:
SaveAreaView is not needed in `block-toolbar` anymore. The one inside `block-manager` takes care of the toolbar.